scripts/cnode-helper-scripts/cntools.library

```diff
@@ -959,7 +959,7 @@ getBalance() {
   asset_name_maxlen=5; asset_amount_maxlen=12
   tx_in=""
   
-  if [[ -z ${KOIOS_API} ]]; then
+  # if [[ -z ${KOIOS_API} ]]; then
```

Comment:
if we're disabling it, might as well remove the section?
Can always look back on previous codebase if desired, IMO - would be good to keep current version clean